src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -78,13 +78,60 @@ private static DeviceDataManager createDeviceDataManager() {
 		return deviceDataManager;
 	}
 
+	private static DataModel createModelSingleBand(String band) {
+		DataModel model = new DataModel();
+
+		final int channel = UCentralUtils.LOWER_CHANNEL_LIMIT.get(band);
+
+		State stateA = TestUtils.createState(
+			channel,
+			DEFAULT_CHANNEL_WIDTH,
+			MAX_TX_POWER,
+			BSSID_A
+		);
+		State stateB = TestUtils.createState(
+			channel,
+			DEFAULT_CHANNEL_WIDTH,
+			MAX_TX_POWER,
+			BSSID_B
+		);
+		State stateC = TestUtils.createState(
+			channel,
+			DEFAULT_CHANNEL_WIDTH,
+			MAX_TX_POWER,
+			BSSID_C
+		);
+
+		model.latestState.put(DEVICE_A, stateA);
+		model.latestState.put(DEVICE_B, stateB);
+		model.latestState.put(DEVICE_C, stateC);
+
+		model.latestDeviceStatus.put(
+			DEVICE_A,
+			TestUtils
+				.createDeviceStatusSingleBand(channel, MAX_TX_POWER)
+		);
+		model.latestDeviceStatus.put(
+			DEVICE_B,
+			TestUtils
+				.createDeviceStatusSingleBand(channel, MAX_TX_POWER)
+		);
+		model.latestDeviceStatus.put(
+			DEVICE_C,
+			TestUtils
+				.createDeviceStatusSingleBand(channel, MAX_TX_POWER)
+		);
+
+		return model;
+	}
+
 	/**
 	 * Creates a data model with 3 devices. All are at max_tx_power, which
 	 * represents the first step in greedy TPC.
 	 *
 	 * @return a data model
 	 */
-	private static DataModel createModel() {
+	private static DataModel createModelDualBand() {
 		DataModel model = new DataModel();
```

Comment:
I think dual band means a band with two channels in it, rather than an AP has two bands. Just want to make sure we are using the correct terminology. @pohanhf

Reply:
Dual band means two separate bands, so this is fine